code/headers/st7735.hpp

```diff
@@ -1,11 +1,13 @@
 #pragma once
 
-#include <hwlib.hpp>
 #include <display_adapter.hpp>
+#include <hwlib.hpp>
+
 
 namespace r2d2::display {
 
-    class st7735_c : public display_c {
+    template <std::size_t CursorCount, class DisplayScreen>
+    class st7735_c : public display_c<CursorCount, DisplayScreen> {
     protected:
         // all the commands for the display
         constexpr static uint8_t _NOP = 0x00;
```

Comment:
Variable or structure names in the form of 'underscore' + 'capital' (e.g. `_C`) are reserved by the standard for the compiler. Please rename these, you could remove the leading underscore.

Reply:
should allready be fixed in the latest version